api/documents/api.py

```diff
@@ -0,0 +1,7 @@
+from ninja import NinjaAPI
+from .views import router as documents_router
+
+api = NinjaAPI()
+
+
+api.add_router("documents", documents_router, tags=["Documents"])
```

Comment:
**nitpick:** since ninja 1.0 you don't have to import routers, just put their python module in quotes in here and it'll automagically work as ninja expands modules under the hood then